lib/pxlogger.js

```diff
@@ -4,27 +4,25 @@
  */
 
 class PxLogger {
-    constructor() {
-        this.debugMode = false;
-        this.appId = '';
-    }
+  constructor(params) {
+    this.debugMode = params.debugMode || false;
+    this.appId = params.pxAppId || "PX_APP_ID";
```

Comment:
Can you please explain why assigning a value (rather than '') when there is no pxAppId in the params object?
By doing it, we cant check if the variable is empty.

Reply:
I'm not sure how to answer this question - [the current default app id is `'PX_APP_ID'`](https://github.com/PerimeterX/perimeterx-node-core/blob/master/lib/pxconfig.js#L194), so I was adhering to that. Do you guys want me to change this behavior to use a blank string as default?

Reply:
Yeah, it's worth making clear that this change is to keep compatibility with the resulting defaults from `PxConfig` now that the circular dependency has been removed. Changing the value to blank would be a change to what people see today.

Reply:
Ok. I missed the fact that this is just keeping the same logic as before.